dialog/internal/dialog.ts

```diff
@@ -238,7 +238,7 @@ export class Dialog extends LitElement {
         .returnValue=${this.returnValue || nothing}
       >
         <div class="container"
-          @click=${this.handleContentClick}
+          @mousedown=${this.handleContentMouseDown}
```

Comment:
Can we swap this to `pointerdown`?

Reply:
And preserving the existing `click` event you mean?

Reply:
I swapped to pointerdown and renamed the flag so it reads better.

Reply:
Doesn't click return the target where mouse up happens?